skrub/tests/test_to_str.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from skrub import _dataframe as sbd
+from skrub._on_each_column import RejectColumn
+from skrub._to_categorical import ToCategorical
+from skrub._to_datetime import ToDatetime
+from skrub._to_str import ToStr
+
+
+def test_to_str(df_module):
+    values = [object(), True, None, "one"]
+    out = ToStr().fit_transform(df_module.make_column("", values))
+    assert sbd.is_string(out)
+    if df_module.name == "pandas":
+        assert out.dtype == "O"
+        assert pd.NA not in out
+    expected = df_module.make_column(
+        "", [None if v is None else str(v) for v in values]
+    )
+    if df_module.description == "pandas-nullable-dtypes":
+        null = expected.isna()
+        expected = expected.astype("str")
+        expected[null] = None
+    df_module.assert_column_equal(sbd.drop_nulls(out), sbd.drop_nulls(expected))
+    df_module.assert_column_equal(sbd.is_null(out), sbd.is_null(expected))
```

Comment:
Are these two lines testing the same thing ?

Reply:
no, the first compares the non-missing values and the second compares the missingness pattern